analytics/upress_reporting/models/pollers/poller.py

```diff
@@ -1,137 +1,118 @@
-import boto3
 import geocoder
-import json
 import os
 import pandas
 import re
 
-from abc import ABC, abstractmethod
-from helpers import aggregate_logs
-from managers.db import DBManager
-from models.data.interaction_event import InteractionEvent, UsageType
+from models.data.interaction_event import InteractionEvent, InteractionType
 
 IP_REGEX = r"\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}"
+REQUEST_REGEX = r"REST.GET.OBJECT "
+TIMESTAMP_REGEX = r"\[.+\]"
 
 
-class Poller(ABC):
-    def __init__(self, publisher, date_range):
-        self.publisher = publisher
+class Poller():
```

Comment:
poller is generic. maybe UsageEventDataPoller is more explicit?

Reply:
I renamed to InteractionEventPoller to stay consistent with the data type name! UsageEventDataPoller seems a little confusing to me but lmk your thoughts

Reply:
that works!